WIP Drawing the power module

imuguruza opened the pull request:
I have started drawing the power module!

imuguruza committed:
Creation of the project

First steps

imuguruza committed:
WIP

keep drawing!

imuguruza committed:
WIP II

INA169 insertion in design

vmayoral commented:
@imuguruza you'll have to create those components that you don't find. Feel free to create a new library for them.

imuguruza committed:
WIP

Inserting more elements.

imuguruza committed:
WIP First preliminar version

Some components need to be replaced, diode mbr120 library must be found.
And some packages are not equal

imuguruza committed:
WIP - First board approach

WIP - Editting the board

imuguruza committed:
WIP- Second layout design

WIP - Trying to figure out a better solution. Schematic error solved

imuguruza committed:
Readme update
